# Fix compilation with Chromaprint 1.4
diff -urNad ffmpeg-3.2.2-orig/libavformat/chromaprint.c ffmpeg-3.2.2/libavformat/chromaprint.c
--- ffmpeg-3.2.2-orig/libavformat/chromaprint.c	2016-12-06 01:28:54.000000000 +0200
+++ ffmpeg-3.2.2/libavformat/chromaprint.c	2016-12-06 11:38:19.200467266 +0200
@@ -39,7 +39,11 @@
     int silence_threshold;
     int algorithm;
     FingerprintFormat fp_format;
+#if CPR_VERSION_INT >= AV_VERSION_INT(1, 4, 0)
+    ChromaprintContext * ctx;
+#else
     ChromaprintContext ctx;
+#endif
 } ChromaprintMuxContext;
 
 static void cleanup(ChromaprintMuxContext *cpr)
